tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/SharedTaggedMetricRegistries.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.tritium.metrics.registry;
+
+import com.codahale.metrics.SharedMetricRegistries;
+import java.util.concurrent.TimeUnit;
+
+/** Analogous to codahale's {@link SharedMetricRegistries}. */
```

Comment:
note, the codahale one has significantly more features - but I'm not sure we really need them or want them? https://github.com/dropwizard/metrics/blob/v3.2.6/metrics-core/src/main/java/com/codahale/metrics/SharedMetricRegistries.java

Reply:
I'm generally supportive of the simple singleton as it is often difficult to inject/plumb through to all the places we actually want to instrument. I don't think we need the complexity of the separately named registries as we implicitly assume the single shared registry for our services.